agent/osv_agent.py

```diff
@@ -48,6 +51,31 @@
     "MACHO_LIBRARY": ["OSS-Fuzz", "Alpine", "Debian", "Linux", "Bitnami", "SwiftURL"],
 }
 
+FILE_TYPE_BLACKLIST = (
+    ".car",
+    ".dex",
+    ".dylib",
+    ".eot",
+    ".gif",
+    ".ico",
+    ".jpeg",
+    ".jpg",
+    ".mobileprovision",
+    ".nib",
+    ".pdf",
+    ".plist",
+    ".png",
+    ".psd",
+    ".so",
+    ".strings",
+    ".svg",
+    ".symbols",
+    ".ttf",
+    ".woff",
+    ".woff2",
+    ".zip",
+    ".xml",
```

Comment:
xml no, some lock files are XML. You need to double check the sbom and lock files list please.

Reply:
I would even suggest whitelisting the support formats instead.